awsmp/models.py

```diff
@@ -221,14 +221,13 @@ class Region(BaseModel):
     @field_validator("commercial_regions")
     def commercial_region_validator(cls, value):
         client = boto3.client("ec2")
-        available_regions = [region["RegionName"] for region in client.describe_regions()["Regions"]]
-        if value[0] == "all":
```

Comment:
We're dropping support for "all" regions here. Do we have that documented or used anywhere that needs to be updated?

Also, not strictly related to this PR, but we're well overdue for a documentation refresh. I don't think entity-diff or download are documented